meinberlin/apps/maptopicprio/views.py

```diff
@@ -156,3 +159,12 @@ def get_success_url(self):
 
     def get_permission_object(self):
         return self.get_object()
+
+    def delete(self, request, *args, **kwargs):
+        response = super().delete(request, *args, **kwargs)
+        a4dashboard_signals.module_component_updated.send(
+            sender=None,
+            module=self.module,
+            component=self.component,
+            user=self.request.user)
+        return response
```

Comment:
Could we extend the DashboardComponentUpdateSignalMixin to handle the delete too?

Reply:
good idea, this would overwrite the DeletionMixin